README.md

```diff
@@ -23,7 +23,7 @@ I was among the users who were deeply affected by the recent change. I still thi
 
 > *I choose to walk in shades.* <br>
 > *Hearken now, to the song of dusk* <br>
-> *The forest venerates your name* <br> 
```

Comment:
Looks like the directory is configured with prettier, so why not run prettier on the README? :thinking: 

It looks like it could have a lot more changes